pkgs/applications/video/kodi/plugins.nix

```
@@ -506,4 +507,64 @@ let self = rec {
     };
   };
 
+  vfs-sftp = mkKodiABIPlugin rec {
+    namespace = "vfs.sftp";
+    version = "1.0.1";
+    plugin = namespace;
+
+    src = fetchFromGitHub {
+      owner = "xbmc";
+      repo = namespace;
+      rev = "${version}-${rel}";
+      sha256 = "1l9igrl168s91c15v9klyaaz226ik3xlbzjk2f1346fvzmp87g9v";
+    };
+
+    meta = with stdenv.lib; {
+      description = "SFTP Virtual Filesystem add-on for Kodi";
+      platforms = platforms.all;
+      maintainers = with maintainers; [ minijackson ];
+    };
+
+    extraBuildInputs = [ openssl libssh zlib ];
+  };
+
+  vfs-rar = mkKodiABIPlugin rec {
+    namespace = "vfs.rar";
+    version = "2.0.6";
+    plugin = namespace;
+
+    src = fetchFromGitHub {
+      owner = "xbmc";
+      repo = namespace;
+      rev = "${version}-${rel}";
+      sha256 = "1p21x1ncj233qa4zsi8nazcdbh0vqr36pabldmbwsp2zd5nzqz0q";
+    };
+
+    meta = with stdenv.lib; {
+      description = "RAR Virtual Filesystem add-on for Kodi";
+      platforms = platforms.all;
```

Found what we might guess the license is at https://github.com/xbmc/vfs.libarchive/blob/2ba11021ef58cf7fa9ca1051e5c0d558cd2a04bc/debian/copyright#L18

I checked and it's the same for all of them.

```suggestion
      license = licenses.gpl2Plus;
      platforms = platforms.all;
```